.github/workflows/continuous-deployment-server.yml

```diff
@@ -0,0 +1,98 @@
+name: Continuous Deployment indy-vdr-proxy-server
+
+on:
+  push:
+    branches: [main]
+
+  workflow_dispatch:
+
+env:
+  DH_USERNAME: ${{ secrets.DOCKER_HUB_LOGIN }}
+  DH_TOKEN: ${{ secrets.DOCKER_HUB_PWD }}
+  IMAGE_NAME: "indy-vdr-proxy-server"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup node v22
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
+
+      - name: Install dependencies
+        run: |
+          cd packages/server
+          yarn install
+
+      - uses: codfish/semantic-release-action@v3
```

Comment:
This action will not work well here, since in this repo we are using another mechanism (release-please) to determine package versions. 

So it should be avoided here.

Reply:
Hi! @genaris solved this thread change to release-please method please check and let me know.